plugins/plugin-chart-echarts/src/Timeseries/controlPanel.tsx

```diff
@@ -301,6 +307,47 @@ const config: ControlPanelConfig = {
           },
         ],
         // eslint-disable-next-line react/jsx-key
+        [<h1 className="section-header">{t('X Axis')}</h1>],
+        ['x_axis_format'],
```

Comment:
Why do we have two `x_axis_format`? I'm surprised this didn't show two controls in your screenshot.